### PR TITLE
Fix bogus 'hidden bytes' metric on ARM

### LINKED ISSUE #8249: Ch on ARM build gives bogus bytes hidden metric
```
[0x004008b0]> s main
[0x004008b0]> Ch 100
[0x004008b0]> pdf @ main | head
  ;-- main:
┌ (fcn) main 1272
│   main ();
│ ; var int local_88h @ sp+0x88
│ ; var int local_8ch @ sp+0x8c
│ ; var int local_90h @ sp+0x90
│ ; var int local_94h @ sp+0x94
│ 0x004008b0 (-1297553060 bytes hidden)
│ 0x00400914      afc0003c       sw zero, 0x3c(fp)
│ 0x00400918      afc00040       sw zero, 0x40(fp)
```

Architecture being analyzed does not make a difference.

### PR DESCRIPTION
**Pull request by toshipiazza**
Fixes #8249

### CODE REVIEW

**Comment by radare** [comment]
Can you add a test for this fix?

File: libr/core/disasm.c [hunk 1]
```diff
@@ -2089,7 +2089,7 @@ static int ds_print_meta_infos(RDisasmState *ds, ut8* buf, int len, int idx) {
 					break;
 				}
 				case R_META_TYPE_HIDE:
-					r_cons_printf ("(%d bytes hidden)", mi->size);
+					r_cons_printf ("(%llu bytes hidden)", mi->size);
```
Comment (radare) [inline libr/core/disasm.c]: %llu is not portable. Use PFMT64d

**Comment by radare** [comment]
ping

File: libr/core/disasm.c (hunk 1 above)
Comment (toshipiazza) [reply libr/core/disasm.c]: Done.

**Comment by toshipiazza** [comment]
Sorry about the delay, where do I add tests?

**Comment by radare** [comment]
See the radare2-regressions repo

**Comment by toshipiazza** [comment]
Will do

**Pull request merged**